warp/cli/warp_cli.py

```diff
@@ -69,12 +67,17 @@ def invoke(program, address, function, inputs):
     required=True,
     type=click.Path(exists=True, dir_okay=False),
 )
-def deploy(program):
+@click.option("--constructor_args", required=False, default="\0")
+def deploy(program, constructor_args):
```

Comment:
@murcake in this case they'll be "\0"

Reply:
I see. I'd prefer handling a default of `None` and have it handled here though.